tools/mo/openvino/tools/mo/front/tf/partial_infer/tf.py

```diff
@@ -2,16 +2,23 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging as log
+import os
 from re import match
 
 import numpy as np
 
+# do not print INFO and WARNING messages from TensorFlow
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
 try:
     import tensorflow.compat.v1 as tf_v1
     # disable eager execution of TensorFlow 2 environment immediately
     tf_v1.disable_eager_execution()
 except ImportError:
     import tensorflow as tf_v1
+
+#in some environment suppressing through TF_CPP_MIN_LOG_LEVEL does not work
```

Comment:
add ` ` after # at the begin of comment line here and in the same lines below.

Reply:
Will fix this in the next PR